hugo.toml

```diff
@@ -306,7 +306,7 @@ i18nDir = "./data/i18n"
 description = "Production-Grade Container Orchestration"
 
 [languages.bn]
-title = "Kubernetes"
+title = "কুবারনেটিস"
```

Comment:
I'm fine with this. Some localization teams use the Roman alphabet original, but that's definitely optional.

We don't have an equivalent to `languageNameLatinScript` for this string, because in the Roman alphabet it will always be `Kubernetes`.

Reply:
I agree Tim. I'm not against the change. iMO, there needs to be uniformity in how we do this across localisations is all. Doing it for some and not doing it for others might cause confusion wrt standardisation. If we propose to do this for the Bengali localisation, my only ask is that we socialize it on the  #sig-docs-localization Slack channel and document it appropriately.

Reply:
We could add a `preferRomanTitle` boolean in case a localization wants that preference, and still needs to record how to write Kubernetes in the target writing system.

Reply:
Yep, so the ask @asem-hamid from you, as a requester, is to have the localisation/localisation leads bubble it up as a conversation item during the monthly meeting for localisation & seek the wider community's inputs.

Reply:
This PR was created in May, and currently, I'm not sure about the motivation behind localizing the title. It could be a misunderstanding on my part. If that's the case, I sincerely apologize.

Also, I can't find any reflection of this change in the deploy preview, and I'm not sure if it actually changes anything on the website.

If this doesn't change anything on the website, I think it might not be necessary to localize the title. I'd like to hear the opinions of the other localization team members on this matter.

 Additionally, I would like to kindly ask the website owners if they notice any changes as a result of localizing this title

Reply:
@asem-hamid, this pull request (PR) localizes the website title to Bengali (bn). As shown in the attached screenshot.

![image](https://github.com/user-attachments/assets/4fdfab6c-e26b-4643-8e82-68d3a54c5b00)

Reply:
Oh, thank you, @sajibAdhi. I totally missed that.

I think it’s a reasonable change for our localized version. Let's see the opinions of the other teams and the leads.

Reply:
@asem-hamid : Please see [this reply](https://github.com/kubernetes/website/pull/46441#discussion_r1721691808). We understand that this changes the title, however, this is **not** standard across localisations and might cause confusion in the future. The ask is to have this bubbled up during the next monthly call for localization, gain consensus and ensure it gets documented properly.

Also, refer to [this comment by Tim](https://github.com/kubernetes/website/pull/46441#discussion_r1721825628) to understand more about other options.

Reply:
@divya-mohan0209 I understand your concern. I'm also waiting for feedback from the other teams and leads at the next monthly meeting. Hope we can  pick up best option there .

Reply:
```suggestion
title = "Kubernetes (কুবারনেটিস)"
```

xref here

> Hello, let me comment as a Japanese l10n team.
> 
> In Japanese, there is no word equivalent to “Kubernetes.” So, our team hasn’t had this kind of discussion.
> However, as long as the entire site is “consistent,” I think each l10n team can make their own decisions.
> (We also often discuss translations of terms like Pod and Node.)
> 
> However, how about using `Kubernetes (কুবারনেটিস)` if you want to change title in k/website?

Reply:
@Okabe-Junya taking this approach . Thank you